backend/apps/accounts/models.py

```
@@ -167,9 +153,29 @@ class UserProfile(models.Model):
+        db_table = "accounts_userprofile"
+
     def __str__(self):
         return f"UserProfile({self.user.username})"
 
+    def increment_jwt_version(self):
+        """
+        Increment JWT token version to invalidate all existing tokens.
+        Called when user changes password.
+        """
+        self.jwt_token_version += 1
+        self.last_password_change = timezone.now()
```

Ruff (0.15.20)

[error] Undefined name `timezone`

(F821)